Data/future/nymex/margins/1S.csv

```diff
@@ -1,11 +1,1823 @@
+20141117,3306,3005
+20141118,3194,2903
+20141119,3078,2798
+20141120,2957,2688
+20141121,2832,2574
+20141124,2415,2195
+20141125,2259,2053
+20141126,2092,1901
+20141127,1909,1735
+20141128,4675,4250
 20141205,8250,7500
-# https://www.cmegroup.com/notices/clearing/2016/09/Chadv16-389.html
+20141208,7841,7128
+20141209,7699,6999
+20141210,7556,6869
+20141211,7409,6735
+20141212,7259,6599
+20141215,6791,6173
+20141216,6627,6024
+20141217,6460,5872
+20141218,6287,5715
+20141219,6110,5554
+20141222,5544,5040
+20141223,5343,4857
+20141224,5133,4666
+20141225,4915,4468
+20141226,4686,4260
+20141229,3921,3564
+20141230,3630,3300
+20150102,8250,7500
+20150105,7980,7254
+20150106,7841,7128
+20150107,7699,6999
+20150108,7556,6869
+20150109,7409,6735
+20150112,6950,6318
+20150113,6791,6173
+20150114,6627,6024
+20150115,6460,5872
+20150116,6287,5715
+20150119,5739,5217
+20150120,5544,5040
+20150121,5343,4857
+20150122,5133,4666
+20150123,4915,4468
+20150126,4191,3810
+20150127,3921,3564
+20150128,3630,3300
+20150129,3314,3012
+20150130,2964,2694
+20150202,8250,7500
+20150204,8102,7365
+20150205,7950,7227
+20150206,7796,7087
+20150209,7313,6648
+20150210,7145,6495
+20150211,6973,6339
+20150212,6796,6178
+20150213,6615,6013
+20150216,6038,5489
+20150217,5834,5303
+20150218,5621,5110
+20150219,5401,4910
+20150220,5172,4701
+20150223,4410,4009
+20150224,4125,3750
+20150225,3820,3472
+20150226,3486,3169
+20150227,3119,2835
+20150302,8250,7500
+20150303,8116,7378
+20150304,7980,7254
+20150305,7841,7128
+20150306,7699,6999
+20150309,7259,6599
+20150310,7106,6460
+20150311,6950,6318
+20150312,6791,6173
+20150313,6627,6024
+20150316,6110,5554
+20150317,5927,5388
+20150318,5739,5217
+20150319,5544,5040
+20150320,5343,4857
+20150323,4686,4260
+20150324,4446,4041
+20150325,4191,3810
+20150326,3921,3564
+20150327,3630,3300
+20150330,2567,2333
+20150331,2096,1905
+20150401,1482,1347
+20150402,8250,7500
+20150406,7971,7246
+20150407,7827,7115
+20150408,7681,6982
+20150409,7532,6847
+20150410,7379,6708
+20150413,6903,6275
+20150414,6737,6124
+20150415,6566,5969
+20150416,6390,5809
+20150417,6211,5646
+20150420,5636,5123
+20150421,5431,4937
+20150422,5218,4743
+20150423,4996,4541
+20150424,4763,4330
+20150427,3986,3623
+20150428,3690,3354
+20150429,3369,3062
+20150430,3013,2739
+20150501,2610,2372
+20150504,7980,7254
+20150505,7841,7128
+20150506,7699,6999
+20150507,7556,6869
+20150508,7409,6735
+20150511,6950,6318
+20150512,6791,6173
+20150513,6627,6024
+20150514,6460,5872
+20150515,6287,5715
+20150518,5739,5217
+20150519,5544,5040
+20150520,5343,4857
+20150521,5133,4666
+20150522,4915,4468
+20150525,4191,3810
+20150526,3921,3564
+20150527,3630,3300
+20150528,3314,3012
+20150529,2964,2694
+20150601,8250,7500
+20150603,8112,7374
+20150604,7971,7246
+20150605,7827,7115
+20150608,7379,6708
+20150609,7224,6567
+20150610,7066,6423
+20150611,6903,6275
+20150612,6737,6124
+20150615,6211,5646
+20150616,6025,5477
+20150617,5834,5303
+20150618,5636,5123
+20150619,5431,4937
```

Comment:
Just some thoughts, no changes required: 

These margins change every day. Maybe this is the percentage of a single contract that is required as initial margin rather than fixed portfolio values. Either way, this is going to be great for backtesting now that we have all of these margins.